libraries/AP_Param/AP_Param.h

```diff
@@ -176,6 +182,7 @@ class AP_Param
             const struct GroupInfo *group_info;
             const struct GroupInfo **group_info_ptr; // when AP_PARAM_FLAG_INFO_POINTER is set in flags
             const float def_value;
+            ptrdiff_t def_value_offset; // Default value offset from param object, when AP_PARAM_FLAG_DEFAULT_POINTER is set in flags
```

Comment:
this costs us 4 bytes of flash per parameter table entry, which is quite expensive. We could union def_value with def_value offset, but would make the code more complex

Reply:
Its in the existing union with def_value and the griopinfo's. Size should be unchanged.